src/main/pg/dyn_notch_config.c

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
The naming of the file seems to be inconsistent with how the existing parameter groups are done - while the data structure name is `...Config`, the file does not carry a `..._config`, so I think naming this `dyn_notch.[ch]` will be more consistent.